test/plugin.test.js

```diff
@@ -0,0 +1,417 @@
+import document from 'global/document';
```

Comment:
This file (previously, `test/videojs-playlist-ui.test.js`) was updated to pass linting.